src/http.ts

```diff
@@ -270,7 +270,10 @@ export class LemmyHttp extends Controller {
   @Security("bearerAuth")
   @Post("/site")
   @Tags("Site")
-  createSite(@Body() form: CreateSite, @Inject() options?: RequestOptions) {
+  async createSite(
```

Comment:
Added async to all these functions (not sure why it wasn't needed in the first place).